gazebo_plugins/src/gazebo_ros_ackermann_drive.cpp

```diff
@@ -209,6 +209,9 @@ void GazeboRosAckermannDrive::Load(gazebo::physics::ModelPtr _model, sdf::Elemen
   // Initialize ROS node
   impl_->ros_node_ = gazebo_ros::Node::Get(_sdf);
 
+  // Get QoS profiles
+  const gazebo_ros::QoS qos = impl_->ros_node_->get_qos();
```

Comment:
If the comment here is addressed https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1091#discussion_r427531640, it would be nice to use `const &` here too.

Same elsewhere.

Reply:
See 32a0b52